app/models/calculated_handover_date.rb

```diff
@@ -63,38 +63,32 @@ def by_offender_ids(offender_id_or_ids)
 
     # Visualization of the calculation:
     #
-    # cad = com allocated date (or start_date in legacy naming)
+    # h = handover date
     # d = days before handover starts that case is considered in upcoming handovers window (e.g. 56 days, or 8 weeks)
     # rd = relative-to date (defaults to "today")
     #
     # "in upcoming handover window":
     #
-    # cad - d                              cad
+    #  h - d                                h
     #   |                                   |
     #   |<----------------------------------|
     #
-    # So, if rd is (cad - d) or later, and rd is less than but not equal to cad, then the
+    # So, if rd is (h - d) or later, and rd is less than but not equal to cad, then the
```

Comment:
Very minor but just for if you remember in a future PR, there's one instance of `cad` not being renamed to `h` here
Not worth a new commit now, but if you remember to change it in future then 👍

Reply:
It definitely is! That was meant to be an explanation of crazy complicated logic so it has to make sense

Reply:
fixed